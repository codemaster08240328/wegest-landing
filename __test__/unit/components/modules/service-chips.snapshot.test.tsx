import React from 'react'
import renderer from 'react-test-renderer'
import ServiceChips from '@/components/modules/service-chips'

const chips: any = [
  {
    id: 'chip0',
    title: 'Testing chips',
  },
]

describe('<ServiceChips> snapshot', () => {
  it('renders correctly when there are no props', () => {
    expect(renderer.create(<ServiceChips />).toJSON()).toMatchSnapshot()
  })

  it('renders correctly when there is className', () => {
    expect(
      renderer.create(<ServiceChips className='class-1' />).toJSON(),
    ).toMatchSnapshot()
  })

  it('renders correctly when chips are passed', () => {
    expect(
      renderer.create(<ServiceChips chips={chips} />).toJSON(),
    ).toMatchSnapshot()
  })
})
