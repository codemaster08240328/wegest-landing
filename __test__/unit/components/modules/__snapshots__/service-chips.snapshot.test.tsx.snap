// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ServiceChips> snapshot renders correctly when chips are passed 1`] = `
<div
  className="makeStyles-root-1 "
  data-testid="chipsContainer"
>
  <div
    className="makeStyles-chips-3"
    data-testid="chips"
  >
    <div
      className="MuiChip-root makeStyles-chip-4 MuiChip-colorSecondary MuiChip-deletableColorSecondary MuiChip-sizeSmall MuiChip-deletable"
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      role="button"
      tabIndex={0}
    >
      <span
        className="MuiChip-label MuiChip-labelSmall"
      >
        Testing chips
      </span>
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root MuiChip-deleteIcon MuiChip-deleteIconColorSecondary MuiChip-deleteIconSmall"
        focusable="false"
        onClick={[Function]}
        viewBox="0 0 24 24"
      >
        <path
          d="M12 2C6.47 2 2 6.47 2 12s4.47 10 10 10 10-4.47 10-10S17.53 2 12 2zm5 13.59L15.59 17 12 13.41 8.41 17 7 15.59 10.59 12 7 8.41 8.41 7 12 10.59 15.59 7 17 8.41 13.41 12 17 15.59z"
        />
      </svg>
    </div>
  </div>
</div>
`;

exports[`<ServiceChips> snapshot renders correctly when there are no props 1`] = `
<div
  className="makeStyles-root-1 "
  data-testid="chipsContainer"
>
  <div
    className="makeStyles-empty-2"
  />
</div>
`;

exports[`<ServiceChips> snapshot renders correctly when there is className 1`] = `
<div
  className="makeStyles-root-1 class-1"
  data-testid="chipsContainer"
>
  <div
    className="makeStyles-empty-2"
  />
</div>
`;
